packages/one-app-runner/README.md

```diff
@@ -0,0 +1,258 @@
+# @americanexpress/one-app-runner
+
+> A command line interface (CLI) tool for running [One App](https://github.com/americanexpress/one-app) locally.
+
+**one-app-runner** allows you to work locally on your [Holocron Module](https://github.com/americanexpress/one-app/blob/master/docs/api/API.md) without having to clone [One App](https://github.com/americanexpress/one-app).
+
+It works by pulling a Docker image for One App and mounting your module to it as a volume.
+
+**Note:** If you are making changes to One App at the same time you are making changes to your Module(s) then you will still need to [clone One App, serve your module to it, and start it yourself](https://github.com/americanexpress/one-app).
+
+## 🤹‍ Usage
+
+Note that [Docker](https://www.docker.com/community-edition#/download) is required and must be installed before **one-app-runner** can be used.
+
+### Quick start
+
+In your Holocron Module directory:
```

Comment:
Should this be in a root module or can it be in any? Does this need clarification?

Reply:
This can be in any module.

Reply:
Clarified in last commit.